### ISSUE TITLE
Add "Settings", "Custom Fields" submenu to each post_type's menu

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#178]
```
What functionality do you want to see?

It'd be cool to have a "settings" menu item under each content-type's menu.  
This makes the settings pages easier to find instead of wandering through the 
CCTM menus.

```

Original issue reported on code.google.com by `fireproofsocks` on 15 Sep 2011 at 4:47

**Comment by GoogleCodeExporter** [comment]
```
See http://codex.wordpress.org/Administration_Menus

<?php add_submenu_page( $parent_slug, $page_title, $menu_title, $capability, 
$menu_slug, $function); ?>

Where: 
$parent_slug='edit.php';
$capability='manage_options';
$menu_slug = 'cctm&a=edit_post_type&pt=xxxx'  // Settings
$menu_slug = 'cctm&a=list_pt_associations&pt=xxxx'  // Custom Fields

```

Original comment by `fireproofsocks` on 15 Sep 2011 at 4:54

**Comment by GoogleCodeExporter** [comment]
```
This has been implemented in 0.9.4
```

Original comment by `fireproofsocks` on 16 Sep 2011 at 3:32
- Changed state: **Fixed**
